### ISSUE TITLE
I can not install version 1.3.9

### ISSUE THREAD

**Issue by sfederl** [pearcec/Invoke-Terraform#24]
**Describe the bug**
When I try to install version 1.3.8 or higher I always get this message:

WARNING: Terraform version 1.3.9 not found.
Exception: Terraform 1.3.9 fail to pass Code Signature test. Uninstalling.

**To Reproduce**
1. install-terraform -tfversion 1.3.8 or higher

**Expected behavior**
Install Terraform version 1.3.8 or higher

**Output**
WARNING: Terraform version 1.3.9 not found.
Exception: Terraform 1.3.9 fail to pass Code Signature test. Uninstalling.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 22H2 Build 22621.14.13
 
**Additional context**

**Comment by pearcec** [comment]
Looks like as of `1.3.8` this is a new thumbprint for the Authenticode.  You can update the thumbprint in your config if you trust it.  I can't find a matching thumbprint on this page. https://www.hashicorp.com/security I suspect they just need to update it.  I emailed security at hashicorp to confirm.

![image](https://user-images.githubusercontent.com/125776/232248694-e7b3eac4-4277-4acd-84bc-117586a8f9ef.png)

**Comment by sfederl** [comment]
Can you help me, how the update the thumbprint for this module?

**Comment by pearcec** [comment]
It would look something like this 

```
Set-TerraformConfiguration -Configuration @{'HashiCorpWindowsThumbprint' = 'thumbprint2' } -Confirm:$false
```

Replace thumbprint2 with the one you trust.

**Comment by sfederl** [comment]
When i add the new Thumbprint then i can install the version 1.3.8 but i can´t use 1.3.7. Can i add the two Thumbprints? 
Or ignore the Thumbprints? 

Thanks for help!

**Comment by pearcec** [comment]
The PR I have right now will allow for a string of thumbprints. I am waiting to hear back from Hashicorp before I publish the update.

**Comment by pearcec** [comment]
Issue should be fixed in 0.6.2. https://www.powershellgallery.com/packages/Invoke-Terraform/0.6.2

**Comment by sfederl** [comment]
Now i can install version 1.3.8 but not was older is as 1.3.7
![image](https://user-images.githubusercontent.com/114988818/232993954-bb634cf9-327a-44a8-8b54-4df7060581f0.png)
![image](https://user-images.githubusercontent.com/114988818/232994341-3de0b1b2-87dd-4970-a11d-4e017c347a7e.png)
